source/Windows.Devices.I2c/I2cController.cs

```diff
@@ -56,33 +55,27 @@ internal static Hashtable DeviceCollection
                 s_deviceCollection = value;
```

Comment:
Maybe I am lacking C# knowledge here, but where does this "value" stem from please?

Reply:
On a "setter" this is the value that is being _set_ for the property.